Code/GraphMol/MolOps.h

```diff
@@ -158,6 +159,8 @@ double computeBalabanJ(double *distMat, int nb, int nAts);
     \param addCoords    (optional) If this is true, estimates for the atomic
    coordinates
                 of the added Hs will be used.
+    \param onlyOnAtoms   (optional) if provided, this should return a vector of
```

Comment:
return a vector -> be a vector

(return makes it sound like onlyOnAtoms is initialized by this addHs)